### ISSUE TITLE
[Accessibility] Card reveal button not accessible by keyboard interaction

### ISSUE THREAD

**Issue by gselderslaghs** [materializecss/materialize#539]
### Before submitting...

- [X] I have searched for [duplicate or closed issues](https://github.com/materializecss/materialize/issues?utf8=%E2%9C%93&q=is%3Aissue).
- [X] I have read the [CONTRIBUTING document](https://github.com/materializecss/materialize/blob/master/CONTRIBUTING.md) and my issue is following the guidelines.
- [X] I have read the template completely before filling it in.

### Context

When using the Card component with reveal modifier, reveal button is not tabbable/focusable by keyboard interaction which causes reduced accessibility towards people who rely on keyboard input

### Current Behavior

When trying to tab into the reveal button, the element is ignored by keyboard interaction

[Codepen example](https://codepen.io/Geert-Selderslaghs/pen/jENqMJZ)

### Expected behavior

Reveal button should be tabbable/focusable by keyboard interaction

### Possible Solutions or Causes

Implement the necessary functional attributes on initialization in order to make options tabbable/focusable by keyboard interaction

### Your Environment

* Version used: 2.1.1

**Comment by gselderslaghs** [comment]
Since v2.2.1 `M.Cards.init(<selector>);` is now necessary for the reveal to work properly, I think we can fix this by initialize cards programmatically on page load, this is also the case for other components, suggest to create a new issue or discussion for this
Closing this for now
